documentation/src/docs/admin-guide/lts-sizing.md

```diff
@@ -106,4 +110,48 @@ the thread pool size to 2-4 threads.
+
+- _Number of Segment Containers (`pravegaservice.container.count`, `controller.container.count`)_: Number of Segment Containers in the system. 
+This value must be the same across all Segment Store instances in this cluster. This value cannot be changed dynamically. 
+See [documentation for details](https://cncf.pravega.io/docs/nightly/segment-store-service/#segment-containers).
+_Type_: `Integer`. _Default_: `4`. _Update-mode_: `read-only`.
+
+- _Replication degree (`bookkeeper.write.quorum.size`)_: Write Quorum size for BookKeeper ledgers.
+_Type_: `Integer`. _Default_: `3`. _Update-mode_: `per-server`.
+
+Each Segment Container has a Durable Log (default implementation is `BookkeeperLog`). For context, a Segment
+Container multiplexes and writes all the operations of active Segments to its Durable Log. In the Bookkeeper implementation
+of the Durable Log, Segment operations are grouped into `DataFrames` and written as individual entries into a Ledger.
+If the Ledger reaches the maximum size specified in `bookkeeper.ledger.size.max`, then the current Ledger is closed and
+a new one is created (i.e., Ledger rollover). In parallel, as soon as data for a Segment Container is correctly persisted
+in long-term storage, the Durable Log can be truncated. When working with Bookkeeper, truncating a Durable Log means
+deleting the Ledgers whose last entry is lower than the last truncation point. Once these Ledgers are marked as deleted,
+Bookkeeper can delete them when running garbage collection, thus freeing up space to continue writing data.
+
+With the above context, the question to answer is what is the minimum amount of storage space that is needed
+for preventing Pravega from exhausting storage space in Bookkeeper during normal operation? We suggest provisioning storage 
+space for at least 2 full-length ledgers per container (so one Ledger can be garbage collected by Bookkeeper while the Segment
+Container continues to write to the new one). Therefore, we could estimate the storage space in Bookkeeper as follows:
+
+_Bookie journal/ledger volume_: 2 * `bookkeeper.ledger.size.max` * `pravegaservice.container.count` * (`bookkeeper.write.quorum.size` / `#bookies`)
+
+That is, we need a storage space on Bookies that can allocate 2 max size ledgers per container. Nonetheless, we have also to take
+into account the replication factor of Pravega writing to Bookkeeper, as well as the number of Bookies. For example, assuming the
+default values, we would need `2 * 4 Segment Containers * 1GB = 8GB` of storage space in Bookkeeper. This is in the
+case of having 3 Bookies (as `bookkeeper.write.quorum.size = 3`). If we had 6 Bookies instead, the minimum storage space
+to be allocated in Bookie volumes would be 4GB. 
+
+As an important point, note that this is the storage space to be allocated _individually to each journal and ledger volumes of Bookies_.
```

Comment:
This confuses me a bit, maybe we could make the breakdown more explicit. 8GB in total volumes? or 8 GB per volume with 2 volumes per bookie? 48 GB in total volumes?